sphinx/pygments_styles.py

```diff
@@ -28,12 +28,12 @@ class SphinxStyle(Style):
     background_color = '#eeffcc'
     default_style = ''
 
-    styles = FriendlyStyle.styles
-    styles.update({
+    styles = {
+        **FriendlyStyle.styles,
```

Comment:
Why is this one needed?

Reply:
The 'styles' attribute on the base class is typed as immutable, so shouldn't be updated in place

Reply:
Perhaps?
```suggestion
    styles = FriendlyStyle.styles | {
```

Reply:
yeah? i'm personally more familiar with the existing notation. The new notation is 3.9+ only, but i guess that doesn't matter since sphinx is now dropping old interpreters fairly aggressively (which i'm for).

I honestly don't mind. They are completely equivalent